sites/funmooc/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- Remove `MARKETING_CONTEXT` and use `WEB_ANALYTICS_*` context instead
```

Comment:
In commit message:
`Richie provides now a web_analytics context > you mean "context variable"

So we do need anymore to the marketing_context > we do not need to add it to `marking_context` anymore